optimization/matRad_daoProjectionFunction.m

```diff
@@ -0,0 +1,60 @@
+function [apertureInfoVec, isConstrActive] = matRad_daoProjectionFunction(apertureInfoVec,limMx,totalNumOfShapes)
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
This function can be deleted because we handle constraints now directly using IPOPT